sdk/cosmos/azure-cosmos/docs/TimeoutAndRetriesConfig.md

```diff
@@ -0,0 +1,31 @@
+## Cosmos DB Python SDK – Timeout configurations and Retry configurations
+
+### Timeout config
+
+The timeout options for the client can be changed from the default configurations with the options below. These are:
+- `Client Timeout`: can be changed by passing the `timeout` option. This option has no default value, and is only present
+if it is passed in by the user. If not present, the other connectivity timeouts will be used.
+- `Request Timeout`: can be changed by passing the `request_timeout` option. Default value is 60s.
```

Comment:
As mentioned above - pretty sure this was the legacy equivalent of connection_timeout, and can probably be removed from documentation entirely in favour of the latter.